.ruby-version

```diff
@@ -1 +0,0 @@
-2.3.0
```

Comment:
I think we should keep it for development (to avoid odd incompatibilities) and possibly add explicit Ruby version tests on travis?